src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -279,10 +282,25 @@ public Task<ImmutableArray<CodeFixCollection>> GetFixesAsync(Document document,
                 cancellationToken.ThrowIfCancellationRequested();
 
                 await AppendFixesOrSuppressionsAsync(
-                    document, span, diagnostics, result, fixer,
+                    document, span, diagnostics, fixAllForInSpan, result, fixer,
```

Comment:
I usually don't like using flag and lambda to have different behavior. especially once code becomes too complex. but if other is okay, I am fine.